tools/Dashboard/DevHome.Dashboard/Views/AddWidgetDialog.xaml.cs

```diff
@@ -57,7 +57,7 @@ public sealed partial class AddWidgetDialog : ContentDialog
         SelectFirstWidgetByDefault();
     }
 
-    private void FillAvailableWidgets()
+    private async void FillAvailableWidgets()

```

Comment:
*Async()

This is called from the `AddWidgetDialog`'s constructor which cannot technically await on this task. To prevent concurrent execution, I suggest subscribing a `RelayCommand` to the Loaded event.

```xaml
    <i:Interaction.Behaviors>
        <ic:EventTriggerBehavior EventName="Loaded">
            <ic:InvokeCommandAction Command="{x:Bind LoadedCommand}" />
        </ic:EventTriggerBehavior>
    </i:Interaction.Behaviors>
```

```cs
[RelayCommand]
public async Task OnLoadedAsync() {...}
```